### ISSUE TITLE
Docker build fails with the error: `failed to create SCM: file does not exist`

### ISSUE THREAD

**Issue by laat** [vercel/turborepo#2296]
### What version of Turborepo are you using?

1.6.0

### What package manager are you using / does the bug impact?

Yarn v1

### What operating system are you using?

Linux

### Describe the Bug

Building examples/with-docker/ fails with the error `failed to create SCM: file does not exist`

turbo looks for a `.git` folder, which is not present in the docker image, and fails [somehow](https://github.com/vercel/turborepo/blob/d18255d7262a826e941cfe6af1d3469f54db1221/cli/internal/scm/scm.go#L48-L54). possibly introduced in 87cfbe9af166991b7b0f5f3603933a4e14f59751

### Expected Behavior

This worked in 1.5.6 (with the libc6-compat package added)

### To Reproduce

Building the [with-docker](https://github.com/vercel/turborepo/tree/main/examples/with-docker) example fails

Fix the examples/with-docker/apps/*/Dockerfile to make turbo run in the installer stage
```patch
 # Add lockfile and package.json's of isolated subworkspace
 FROM node:alpine AS installer
+RUN apk add --no-cache libc6-compat
 RUN apk update
 WORKDIR /app
```

```sh
cd examples/with-docker
yarn install # generates the required yarn.lock
```

Perform the documented build step in the example
```sh
# Build prod using new BuildKit engine
COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose -f docker-compose.yml build --parallel
```

And the build fails like this:
```
#0 0.531 $ /app/node_modules/.bin/turbo run build --filter=web...
#0 0.610  ERROR  run failed: failed to create SCM: file does not exist
```

**Comment by coinlordd** [comment]
I can confirm this isn't limited to Docker, got the same issue with a normal build. Also only happens to 1.6.0, downgrading to 1.5.6 worked again.

**Comment by joelgarrett** [comment]
Also started experiencing the same issue in docker today, can confirm downgrading does fix the problem.

**Comment by nathanhammond** [comment]
Hey y'all, we're on it!

**Comment by nathanhammond** [comment]
This change is released in 1.6.1!

**Comment by laat** [comment]
Thanks! 1.6.1 works for me
